proto/money/folio/folio_state.proto

```diff
@@ -8,14 +8,17 @@ import "primitive/monetary_amount.proto";
 import "money/folio/posted_card_authorization.proto";
 import "money/folio/posted_charge_credit.proto";
 import "money/folio/posted_payment.proto";
+import "money/folio/posted_refund.proto";
 
 message FolioState {
 	repeated PostedChargeCredit charge_credits = 1;
 	repeated PostedCardAuthorization card_authorizations = 2;
 	repeated PostedPayment payments = 3;
+	repeated PostedRefund refunds = 4;
```

Comment:
In general I wouldn't recommend doing "inserts" here. This breaks the new proto object's ability to parse the old ones. Not a biggy for GRPC records, but it's something we'll need to be very cognizant of for event objects.

Feel free to leave this as-is for now.